source/docs/advanced/features/debugging.md

```diff
@@ -0,0 +1,88 @@
+---
+id: debugging
+title: Debugging
+description: Debug your Front-Commerce app.
+---
+
+# Debugging
+
+Debugging your code is an important part of the development process. This documentation explains how you can debug your Front-Commerce frontend and backend code using either the [VS Code debugger](https://code.visualstudio.com/docs/editor/debugging) or [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools).
+
+Any debugger that can attach to Node.js can also be used to debug a Front-Commerce application. You can find more details in the Node.js [Debugging Guide](https://nodejs.org/en/docs/guides/debugging-getting-started/).
+
+## Debugging with VS Code
+
+Create a file named `.vscode/launch.json` at the root of your project with the following content:
+
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "FC: debug server-side",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run start -- --inspect"
+    },
+    {
+      "name": "FC: debug client-side",
+      "type": "pwa-chrome",
+      "request": "launch",
+      "url": "http://localhost:4000"
+    },
+    {
+      "name": "FC: debug full stack",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "npm run start -- --inspect",
+      "console": "integratedTerminal",
+      "serverReadyAction": {
+        "pattern": "started server on .+, url: (https?://.+)",
+        "uriFormat": "%s",
+        "action": "debugWithChrome"
+      }
+    }
+  ]
+}
+```
+
+If you're [changing the port number](/docs/reference/environment-variables.html#Host) your application starts on, replace the `3000` in `http://localhost:3000` with the port you're using instead, you can use the the url defined in your `FRONT_COMMERCE_URL`
```

Comment:
Is it a typo?
```suggestion
If you're [changing the port number](/docs/reference/environment-variables.html#Host) your application starts on, replace the `4000` in `http://localhost:4000` with the port you're using instead, you can use the the url defined in your `FRONT_COMMERCE_URL`
```

Reply:
Hehe yea typo, been years that I work with `3000` as the default 😬